cosign.pub

```diff
@@ -1,4 +1,4 @@
 -----BEGIN PUBLIC KEY-----
-MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEk/2hQrKLFNf66dOyCEoUCip38EZh
-I4NQN1BiVQ+6IRXkHCJahZfPtC9RqJ3dpGm/2CSdjKyES+vDQrUFj8vsmg==
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEBtCtXTn6bzk4V7Na9Yx2IF/xZq3M
+xR99zwqesOTrdric9fTLGxTWjSOiM+KA0IHwMODGaeiO/dysRr9vNKDhDQ==
 -----END PUBLIC KEY-----
```

Comment:
When you're done testing, you'll want to remove these changes to the public key file.

Reply:
Fixed in 2808e25c360dd45faf7b00ccb07c008ef089bbd2